src/exec/command.rs

```diff
@@ -11,6 +11,8 @@ struct Outputs {
     stdout: Vec<u8>,
     stderr: Vec<u8>,
 }
+
+/// Used to print the results of commands that run on results in a thread-safe way
```

Comment:
Do weweant to print output from the filter command, or just suppress it?

Reply:
I agree it would make more sense for the filter command to simply reduce the results that are matched before they're output to the user as usual.